kolibri/plugins/management/assets/src/vue/content-page/task-status.vue

```diff
@@ -0,0 +1,95 @@
+<template>
+
+  <div v-if="showProgress" class="task">
+    <h1>
+      {{title}}
+    </h1>
+    <progress max="1" value={{taskProgress}}></progress>
+    <h2>
+      {{subTitle}}
+    </h2>
+    <button class="buttons" @click='hideProgressBar'>
+      {{buttonMessage}}
+    </button>
+    </div>
+
+</template>
+
+
+<script>
+
+  const actions = require('../../actions');
+
+  module.exports = {
+    components: {
+    },
+    $trNameSpace: 'contentPage',
+    $trs: {
+      buttonConfirm: 'Confirm',
+      buttonCancel: 'Cancel',
+      failed: 'Please confirm and try again.',
+      completed: `Please confirm to add channels to 'My Channels' list.`,
+      loading: 'Please wait...',
+
+    },
+    data: () => ({
+      showProgress: true,
```

Comment:
Seems like this shouldn't be necessary: the component is already hidden by the parent when `tasks.length !== 0`. We should avoid local state here if possible.

(If this was meant to be temporary until the delete task API works, let's add comments indicating that.)